src/Uno.UI/UI/Xaml/Media/Matrix.cs

```diff
@@ -7,61 +7,61 @@ namespace Windows.UI.Xaml.Media
 {
 	public struct Matrix : IFormattable
 	{
-		private Matrix3x2 _inner;
+		internal Matrix3x2 Inner;
 
 		public Matrix(double m11, double m12, double m21, double m22, double offsetX, double offsetY)
 		{
-			_inner = new Matrix3x2((float)m11, (float)m12, (float)m21, (float)m22, (float)offsetX, (float)offsetY);
+			Inner = new Matrix3x2((float)m11, (float)m12, (float)m21, (float)m22, (float)offsetX, (float)offsetY);
```

Comment:
don't like having type change each time.

Reply:
This is used only when we use a `MatrixTransform`, so it's not a performance concern.